README.md

```diff
@@ -10,57 +9,86 @@ What Currently Works
+### Q: What is the recommended requirements?
+- Debian or Ubuntu based distro. Fedora and friends may work, but please, YMMV.
+- Python 3.x and knowledge about virtualenv (setup and activate).
+- Internet connection to download the DMM assets.
 
-There aren't actually that many things you can build a web application on, and most of them were out from the start.
+Protip: Use a VM (VMWare, VirtualBox). If you mess up, simply hose it and start squeaky clean. Or you can use an ARM based linux box (R-Pi, ODROID, etc).
 
-* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/) that's insanely unwieldly, performs about as well as a football team in Antarctica and is a nightmare to maintain.  
-  I've written large applications in PHP before, and I'm not going there again.
-
-* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well, preferably without me having me hire another server I can't pay for.  
-  I run a JIRA (Java) and a Discourse (Ruby) instance, and they both chew up nearly everything their servers have to give.
+### Q: Why is it built on Python?
+There aren't actually that many things you can build a web application on, and most of them were out from the start.  
+There was a large rant by one the kcsrv owner, I (SoftwareGuy) who improved this FAQ have condensed it down to make it less ranty.
 
+* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/).
+* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well. Ruby / Java seem to eat the server alive.
 * NodeJS is actually a good candidate, mostly thanks to being blazing fast.  
-  Unfortunately, nearly everything around it (including the runtime itself) is in beta, and many things (such as database abstraction layers) simply aren't mature enough for me to want to depend on it - I have yet to see a robust migration system, for instance.  
-  One day I might even port over at least parts of the server to it (the game API comes to mind), but not right now.
+  Unfortunately, nearly everything around it (including the runtime itself) is in beta.
 
-Python isn't the fastest thing around, but both it and its ecosystem are very stable and mature. SQLAlchemy + Alembic alone makes an incredibly robust database abstraction layer, that nothing I've seen in NodeJS can match. (Do feel free to prove me wrong though.)
+In short, Python isn't the fastest thing around, but both it and its ecosystem are very stable and mature. SQLAlchemy + Alembic alone makes an incredibly robust database abstraction layer.
 
 
 Development Setup
 ---
-
-Probably only works on Linux. Screw Ansible/Vagrant, too hard for me.
+Make sure you have postgresql, postgresql development libs, python3, python3-pip python3-requests, libcurl-openssl-dev and virtualenv packages installed. Consult your package manager for those packages to be installed (yum, apt-get...)
```

Comment:
Unfortunately, there aren't really any other sane backends.

* MySQL is [notoriously awful](http://grimoire.ca/mysql/choose-something-else), with the most important piece of incompetent nonsense being the lack of Transactional DDL… which means a failed migration (for whatever reason) may leave the database in an inconsistent state.
* SQLite is a good second choice, but it sometimes behaves oddly, and lacks some more advanced features. It'll also slow down with a lot of data, not that a development server is really in any danger of hitting that point.
* lol MS SQL and Oracle, I hope you like enterprise nonsense.